ui/pages/token-allowance/token-allowance.js

```diff
@@ -491,6 +494,11 @@ export default function TokenAllowance({
           </Box>
         </Box>
       ) : null}
+      {!isFirstPage && isHardwareWalletConnected && (
+        <Box paddingLeft={2} paddingRight={2}>
+          <LedgerInstructionField showDataInstruction />
+        </Box>
+      )}
```

Comment:
Hey will be nice to have test coverage for above.

Reply:
Seconded.

Reply:
Added here: ce80c0e8be84def35a07a12f42a8c467bf29500f